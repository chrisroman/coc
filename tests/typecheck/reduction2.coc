((\d: [a: *][b: a]a)d (\a:*)(\b:a)b)
