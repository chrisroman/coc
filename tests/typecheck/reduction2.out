((λd: [a: *] [b: a] a) d (λa: *) (λb: a) b)
typecheck_single_term ((λd: [a: *] [b: a] a) d (λa: *) (λb: a) b)
Type: [a: *] [b: a] a
Value: (λa: *) (λb: a) b
